### PR TITLE
Improve parameter documentation

### PR DESCRIPTION
**Pull request by githtz**
Improve the documentation of parameters when using the local postfix docker-container.

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -66,7 +66,7 @@
 # root@localhost user password. Default password is changeme
 #OTRS_ROOT_PASSWORD=changeme
 
-# Sets the container's hostname (auto-generated if not defined).
+# Sets the container's hostname (auto-generated if not defined). Should be defined manually when using local postfix docker-container.
```
Comment (juanluisbaptiste) [inline .env.example:69]: This is not true, this parameter is completely optional.

File: .env.example [hunk 2]
```diff
@@ -126,8 +126,8 @@
 # Port address of the SMTP server to use.
 # OTRS_SMTP_PORT=
 
-# Username to authenticate with.
+# Username to authenticate with. Not needed when using the local postfix docker-container.
```
Comment (juanluisbaptiste) [inline .env.example:129]: Your comment is confusing, marking it as optional is enough.

File: .env.example [hunk 3]
```diff
@@ -126,8 +126,8 @@
...
 # OTRS_SMTP_USERNAME=
 
-# Password of the SMTP user.
+# Password of the SMTP user. Not needed when using the local postfix docker-container.
```
Comment (juanluisbaptiste) [inline .env.example:132]: The same as the previous one, marking it as optional is enough.

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,9 @@
 # docker-otrs Change Log
 
+## 6.0.30 - 2021-05-11
```
Comment (juanluisbaptiste) [inline CHANGELOG.md:3]: I don't understand why this changes are part of your PR...

File: otrs/Dockerfile [hunk 5]
```diff
@@ -1,7 +1,7 @@
 #OTRS ticketing system docker image.
 FROM centos:7
 LABEL maintainer="Juan Luis Baptiste <juan@juanbaptiste.tech>"
-ENV OTRS_VERSION=6.0.33-01
+ENV OTRS_VERSION=6.0.34-02
```
Comment (juanluisbaptiste) [inline otrs/Dockerfile:4]: I don't understand why this changes are part of your PR...

**Review by juanluisbaptiste — COMMENTED** [review]
Hi,

Please use the PR template that was presented to you when creating this PR, is there for a reason. Your changes are not clear and are mixed with other commits that should not be part of this PR.

**Comment by juanluisbaptiste** [comment]
Closing this, create a new PR and explain the intention of your modifications first as asked by the template and answering all of its questions.

**Comment by juanluisbaptiste** [comment]
Forget about the template, I was confused with another repo, but still please explain them and use a better wording, as they are now are confusing.

File: .env.example (hunk 1 above)
Comment (githtz) [reply .env.example:69]: I've had an issue when this parameter was not set. I'll investigate once more.

File: .env.example (hunk 2 above)
Comment (githtz) [reply .env.example:129]: When this parameter was set, the authentication with the local posfix was not successful. Afer removing this and the password, it worked just fine.

File: CHANGELOG.md (hunk 4 above)
Comment (githtz) [reply CHANGELOG.md:3]: I have no idea. I didn't add those lines, obviously. Maybe Github shows the diff between the commit my branch is based on, and the final result after merging my branch into the latest branches commit?
